utf8.c

```diff
@@ -95,10 +95,14 @@ static Janet jutf8_from_codepoints(int argc, Janet *argv) {
   utf8proc_ssize_t sl = l * 4;
   utf8proc_ssize_t bytes_converted = 0;
   const utf8proc_ssize_t alloca_limit = 64;
-  utf8proc_uint8_t *s = l > alloca_limit ? janet_smalloc(l) : alloca(l);
+  utf8proc_uint8_t *s = sl > alloca_limit ? janet_smalloc(sl) : alloca(sl);
   for (utf8proc_ssize_t i = 0; i < l; i++) {
     utf8proc_uint8_t r[4];
-    utf8proc_int32_t cp = janet_unwrap_integer(c[i]);
+    utf8proc_int32_t cp;
+    if (janet_checktype(c[i], JANET_NUMBER)) 
```

Comment:
Minor style thing I would prefer:

```
if (!janet_checktype(...))
  janet_panicf(...);
cp = janet_unwrap_integer(...);
```

Reply:
Indeed, I am on it. It makes sense. Thank you for checking.